app/controllers/application_controller.rb

```diff
@@ -1,2 +1,10 @@
 class ApplicationController < ActionController::API
+  skip_before_filter :verify_authenticity_token
```

Comment:
Hmm. If this filter is still enabled, I wonder if we can find somewhere in the config to disable it and any default filters like it. I don't think we'll need them for an API-only rails server.